pkg/cloudprovider/aws/cloudprovider.go

```diff
@@ -200,3 +223,11 @@ func getCABundle(ctx context.Context) *string {
 	logging.FromContext(ctx).Debugf("Discovered caBundle, length %d", len(transportConfig.TLS.CAData))
 	return ptr.String(base64.StdEncoding.EncodeToString(transportConfig.TLS.CAData))
 }
+
+func (c *CloudProvider) getProvider(ctx context.Context, providerRefName string) (*v1alpha1.AWS, error) {
+	var ant awsv1alpha1.AWSNodeTemplate
+	if err := c.kubeClient.Get(ctx, types.NamespacedName{Name: providerRefName}, &ant); err != nil {
+		return nil, fmt.Errorf("retrieving provider reference %w", err)
```

Comment:
Typical error annotation uses a comma before the next piece of the error. Similarly, I'd keep the concepts lined up for simplicity. Customers are already aware of `providerRef` since it's part of the API, so better to make it apparent that it's the same concept. Similarly, `get` means something, so might as well reuse the concept.
```suggestion
		return nil, fmt.Errorf("getting providerRef, %w", err)
```